recipes-bsp/u-boot/u-boot-iot2050.inc

```diff
@@ -95,6 +95,7 @@ dpkg_build_sstate_finalize_append() {
 
 do_deploy() {
     cp -f "${S}/${SPI_FLASH_IMG}" "${DEPLOY_DIR_IMAGE}/${SPI_FLASH_DEPLOY_IMG}"
+    cp -f "${S}/u-boot-initial-env" "${DEPLOY_DIR_IMAGE}/"
```

Comment:
You also need to update sstate caching.

Reply:
HI jan
I have no expericing how to modify this.
and for the deploy stage, do there need to add to the sstate caching?

Reply:
Just study the code a few lines above and/or the commit that added the already existing line. And, yes, it's mandatory to not break local builds.

Reply:
has modified, pls help to review